tests/examples/test_baleen.py

```diff
@@ -106,7 +108,7 @@ def gold_passages_retrieved(example, pred, trace=None):
 
 # @pytest.mark.slow_test
 # TODO: Find a way to make this test run without the slow hotpotqa dataset
-def _test_compiled_baleen():
+def test_compiled_baleen():
```

Comment:
Isn't it going to break things to have a unittest using OpenAI like this?

Reply:
With the other change that I added in another file this should run based on the cache of the notebook.